shell/common/rasterizer.cc

```diff
@@ -22,6 +22,8 @@ namespace flutter {
 // used within this interval.
 static constexpr std::chrono::milliseconds kSkiaCleanupExpiration(15000);
 
+static const int kDefaultScreenShotThreadMergeLeaseDuration = 10;
```

Comment:
I'd rather the rasterizer be unaware of any thread merging and keep this in view embedder.

Reply:
Updated per our offline discussion.